### DISCUSSION TITLE
What components of `mol-plugin-ui` can be used in a third party application?

### DISCUSSION THREAD

**Discussion by jbrouw2** [molstar/molstar#117] [Q&A]
Hello!

According to the ReadMe, the module `mol-plugin-ui` has components that can be used outside the main application. 
What components are these, and how would they be integrated into a third party application? 

I've imported the molstar npm package to my project and tried importing a number of the exported react components, but none are compiling and rendering. Each exported component seems to require functionality from the others.  Are there any "root" components in the `mol-plugin-ui` module that can be imported and used in another react component? 

Thanks for any help! 
This repo is fantastic and I'm eager to work with it.

**Comment by dsehnal** [comment]
Hi,

this refers to the ability to mount some of the UI components anywhere on the page.

An example of that is https://github.com/molstar/molstar/blob/master/src/examples/proteopedia-wrapper/index.html#L66 and to mount the controls https://github.com/molstar/molstar/blob/master/src/examples/proteopedia-wrapper/ui/controls.tsx#L14

David
